### ISSUE TITLE
v6.1.7 breaks create-react-app production build (and dev too)

### ISSUE THREAD

**Issue by ddolcimascolo** [i18next/i18next-browser-languageDetector#267]
## 🐛 Bug Report

v6.1.7 breaks CRA production build with `Module not found: Error: Default condition should be last one`. Dev server is broken too with the same error

## To Reproduce

npx create-react-app i18next-browser-languagedetector-6.1.7
cd i18next-browser-languagedetector-6.1.7/
npm i --save -E i18next-browser-languagedetector@6.1.7

Just import the module anywhere
```diff
--- a/src/index.js
+++ b/src/index.js
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import i from 'i18next-browser-languagedetector';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
```

npm run build

## Expected behavior

Build succeeds

## Your Environment

- *runtime version*: Node 16
- *i18next version*: Latest
- *os*: Ubuntu 20.04
- *any other relevant information*

**Comment by adrai** [comment]
https://github.com/i18next/i18next-browser-languageDetector/issues/266

**Comment by adrai** [comment]
@gunters63 can you also check this please?

**Comment by ddolcimascolo** [comment]
Reproduction added, though it's really just importing the module :sweat:

**Comment by gunters63** [comment]
The new types entry which fixed issue #266 should be moved to the start of its export entry.
The default entry should always be last, as the error says.

**Comment by adrai** [comment]
v6.1.8 should fix this
